### PR TITLE
Tweak form validation snippet.

### PR DESCRIPTION
**Pull request by XhmikosR**
Non-whitespace diff: <https://github.com/twbs/bootstrap/pull/29359/files?w=1>
Preview: <https://deploy-preview-29359--twbs-bootstrap.netlify.com/docs/4.3/examples/checkout/>

### CODE REVIEW

File: site/content/docs/4.3/examples/checkout/form-validation.js [hunk 1]
```diff
@@ -2,20 +2,18 @@
...
 
-    // Loop over them and prevent submission
-    Array.prototype.filter.call(forms, function (form) {
-      form.addEventListener('submit', function (event) {
-        if (form.checkValidity() === false) {
-          event.preventDefault()
-          event.stopPropagation()
-        }
+  // Loop over them and prevent submission
+  Array.prototype.filter.call(forms, function (form) {
```
Comment (Johann-S) [inline site/content/docs/4.3/examples/checkout/form-validation.js]: Can you remove filter and use `forEach` instead ? There is no filtering inside...

File: site/content/docs/4.3/examples/checkout/form-validation.js [hunk 2]
```diff
@@ -2,20 +2,18 @@
...
+    form.addEventListener('submit', function (event) {
+      if (form.checkValidity() === false) {
```
Comment (Johann-S) [inline site/content/docs/4.3/examples/checkout/form-validation.js]: `if (!form.checkValidity()) {`

**Comment by XhmikosR** [comment]
@Johann-S: those should happen in a separate PR. Feel free to tackle it later.

**Comment by Johann-S** [comment]
Too bad to handle that later when we are able to fix that now

**Comment by XhmikosR** [comment]
It's a different kind of change. It should be done separately.

**Comment by Johann-S** [comment]
Yep, but different commits can be a good solution to fix that with just one PR

**Review by Johann-S — APPROVED** [review]

**Pull request merged**